### PR TITLE
docs(security): journalist-perspective threat model

### PR DESCRIPTION
**Pull request by aaronbrezel**
## Summary

- Adds `docs/threat_models/ssi-toolkit-threat-model-journalist.md` — a threat model from the journalist perspective covering bad outcomes across the full reporting workflow (problem assessment through publication)
- Structured around consequence tags (Journalistic Harm, Operational Waste, Compliance/Ethics Risk) and mitigation strategies (Prevent, Reduce, Educate, Accept) rather than the attacker-surface framing of the tool-level threat model
- Covers 18 bad outcomes (BO01–BO18) including prompt bias, non-determinism, inference budget blindness, and unspecified output format

**Merge order:** depends on the initial threat model PR merging first.

## Manual QA

> N/A — targets `develop`, no runtime behavior changes

## Security

- [x] None of the above — no threat model update needed (this PR extends threat model documentation)

### CODE REVIEW

File: docs/threat_models/ssi-toolkit-threat-model-journalist.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+
+---
+
+## Phase 0 — Problem Assessment
+*Deciding whether and how to use the tool*
+
+| ID | Bad Outcome | Tags | Importance | Mitigation | Description |
+| --- | --- | --- | --- | --- | --- |
+| BO01 | Poor problem decomposition | OW JH | High | Educate, Reduce | The reporter cannot translate their reporting question into discrete computational steps. They ask the AI to do too much in one pass, struggle to identify which steps are inference-appropriate vs. spreadsheet-appropriate, and end up with outputs too broad to act on. |
+| BO02 | Wrong tool for the job | OW | Low | Educate | The reporter reaches for SSI on a task better suited to NotebookLM, Gemini chat, a database query, or a simple regex. They get mediocre results from a tool optimized for something else. |
```
Comment (arbitraryconstants) [inline docs/threat_models/ssi-toolkit-threat-model-journalist.md]: From my perspective this would also be CR (Compliance / Ethics Risk). This tool may not be appropriate for some highly sensitive data, right?

File: docs/threat_models/ssi-toolkit-threat-model-journalist.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+| BO03 | AI over-reliance (manual or expert would be better) | OW JH | Medium | Educate | The reporter uses AI where manual review or a subject-matter expert would be faster, cheaper, and more reliable. They trust the tool's authority over their own or a colleague's judgment. |
+
+---
+
+## Phase 1 — Data Preparation
+*What goes into the tool*
+
+| ID | Bad Outcome | Tags | Importance | Mitigation | Description |
```
Comment (arbitraryconstants) [inline docs/threat_models/ssi-toolkit-threat-model-journalist.md]: What is the `Importance` column in these tables optimizing for? It's not clear to me from this doc.

For instance, "Sensitive data exposure" is Low, which is surprising to me. 

Could be worth revisiting importance meaning and judgments with input from editorial and legal?

File: docs/threat_models/ssi-toolkit-threat-model-journalist.md [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+| BO18 | Unspecified output format | JH OW | High | Prevent, Reduce, Educate | The reporter doesn't constrain the output shape in their prompt. They get free-form paragraphs when they needed a categorical value, a number, or a yes/no. The run can't be aggregated, filtered, or compared downstream — making it analytically useless even if the underlying reasoning was sound. |
+
+---
+
+## Phase 3 — Execution
+*Running the batch and watching it go*
+
+| ID | Bad Outcome | Tags | Importance | Mitigation | Description |
+| --- | --- | --- | --- | --- | --- |
+| BO17 | Inference budget blindness | OW CR | High | Prevent | The reporter doesn't realize that each row is a paid API call. They iterate prompts on large datasets, re-run batches while debugging, or run the full dataset before sampling to validate — burning through budget without a cost estimate or confirmation step. Compounds with BO09 (prompt overfitting), where many full runs happen during iteration. |
```
Comment (arbitraryconstants) [inline docs/threat_models/ssi-toolkit-threat-model-journalist.md]: Consider rephrasing to something like: Lack of awareness of inference budget.

File: docs/threat_models/ssi-toolkit-threat-model-journalist.md [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+
+| ID | Bad Outcome | Tags | Importance | Mitigation | Description |
+| --- | --- | --- | --- | --- | --- |
+| BO12 | Insufficient validation strategy | JH | High | Educate, Reduce | The reporter doesn't validate output in a way appropriate for their specific use case. This includes too-small or non-random spot-checks, but also failing to ground AI-extracted output against source material (e.g., cross-checking inference results against OCR'd text). They've done *a* review but not the right one. (See BO13 for hallucination-specific failures; BO14 for error-rate blindness.) |
+| BO13 | Hallucination acceptance | JH | High | Educate, Accept | The reporter treats AI output as authoritative fact rather than probabilistic inference. Fabricated names, dates, citations, or characterizations pass through unchallenged because they look plausible. |
+| BO14 | False positive/negative blindness | JH | Medium | Educate, Reduce | The reporter validates that some outputs look right but doesn't characterize the error rate. They can't tell editors how reliable the findings are, can't size their verification burden, and don't know if a 15% false positive rate would change their conclusions. |
```
Comment (arbitraryconstants) [inline docs/threat_models/ssi-toolkit-threat-model-journalist.md]: Consider rephrasing to something like: Lack of awareness of false positives/negatives.

File: docs/threat_models/ssi-toolkit-threat-model-journalist.md [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+| --- | --- | --- | --- | --- | --- |
+| BO17 | Inference budget blindness | OW CR | High | Prevent | The reporter doesn't realize that each row is a paid API call. They iterate prompts on large datasets, re-run batches while debugging, or run the full dataset before sampling to validate — burning through budget without a cost estimate or confirmation step. Compounds with BO09 (prompt overfitting), where many full runs happen during iteration. |
+| BO10 | Non-determinism not understood | JH | Medium | Educate | The reporter doesn't realize the same prompt on the same data can produce different results across runs. They treat a single pass as definitive, don't run a reproducibility check, and can't explain to an editor why the numbers shifted when they re-ran it. |
+| BO11 | Scale behavior differences | JH | Medium | Educate, Reduce | The reporter tests on 10 rows, runs on 10,000. They don't verify behavior holds at scale or spot-check a representative sample of the full run. Systematic errors on edge cases (rare formats, unusual values) are invisible until after publication. |
+
+---
+
+## Phase 4 — Evaluation & Validation
```
Comment (arbitraryconstants) [inline docs/threat_models/ssi-toolkit-threat-model-journalist.md:76]: re: BO12-BO14: Eventually, I'd recommend linking out to documentation (once it exists), since none of this stuff is obvious:
- best practices for validating results
- defining what is a spot check
- what makes spot checks statistically significant
- defining error rates of interest
- what makes a "good" rate? What rate is a "bad" rate?

**Review by arbitraryconstants — COMMENTED** [review]
Having this separate doc for Journalist-facing bad outcomes makes sense to me 👍

File: docs/threat_models/ssi-toolkit-threat-model-journalist.md (hunk 1 above)
Comment (aaronbrezel) [reply docs/threat_models/ssi-toolkit-threat-model-journalist.md]: That's a good point. I _think_ that might be covered in BO04?

File: docs/threat_models/ssi-toolkit-threat-model-journalist.md (hunk 2 above)
Comment (aaronbrezel) [reply docs/threat_models/ssi-toolkit-threat-model-journalist.md]: I think you're right. I think I originally meant this to be "Importance to the product roadmap". But I don't think this column is helping a lot right now. 

My vote would be to delete it for now. We can certainly let editorial and legal star BO's in the future using a different method

File: docs/threat_models/ssi-toolkit-threat-model-journalist.md (hunk 1 above)
Comment (arbitraryconstants) [reply docs/threat_models/ssi-toolkit-threat-model-journalist.md]: Mmm, yeah it's covered by BO04 phase 1, but you could eliminate the tool as an option in Phase 0 if CR was considered as part of BO02. 

> ## Phase 0 — Problem Assessment
> *Deciding whether and how to use the tool*

Q: Is this a tool appropriate for handling highly sensitive data? 
A: No.
=> This is the wrong tool for the job. Don't bother moving onto phase 1.

File: docs/threat_models/ssi-toolkit-threat-model-journalist.md (hunk 2 above)
Comment (arbitraryconstants) [reply docs/threat_models/ssi-toolkit-threat-model-journalist.md]: Sounds good ^^

File: docs/threat_models/ssi-toolkit-threat-model-journalist.md (hunk 1 above)
Comment (arbitraryconstants) [reply docs/threat_models/ssi-toolkit-threat-model-journalist.md]: I see your point though.

**Review by arbitraryconstants — DISMISSED** [review]
I'm glad that this journalist-perspective doc exists as well 👍 Good stuff.

File: docs/threat_models/ssi-toolkit-threat-model-journalist.md (hunk 1 above)
Comment (aaronbrezel) [reply docs/threat_models/ssi-toolkit-threat-model-journalist.md]: I agree with you though, this point really belongs in Phase 0. Going to move it up into the higher section
